package.json

```diff
@@ -17,18 +17,24 @@
     ]
   },
   "dependencies": {
+    "@radix-ui/react-accordion": "^1.1.2",
     "autoprefixer": "10.4.14",
+    "class-variance-authority": "^0.7.0",
     "classnames": "^2.3.2",
+    "clsx": "^2.0.0",
```

Comment:
We already use `classnames` to generate dynamic classNames

Reply:
@adirkandel 
I've coded the whole thing (maintainers page) in one day and hadn't took up the time to check the current packages state.
Can you make the proper adjustments that we can omit this package?

Reply:
Issue resolved in https://github.com/Maakaf/maakaf-website/pull/96/commits/3f6e3c5b29f58b9f8bb481a633a5dee04c2ab849 commit